### DISCUSSION TITLE
What is the reason for TagGroup to be role="grid" with rows and gridCells?

### DISCUSSION THREAD

**Discussion by JurajKruzliak** [adobe/react-spectrum#7443] [Q&A] [answered]
Based on few things I've read, it seems to be that it could very well be a role="list", Is there any specific reason that it is grid?

I am wondering because we need to make basically just a tag group that has only the selectable option, to allow one or many to be selected. This could be also achieved basically with ListBox, but as per your documentation that is supposed to be used only with menu and similar style interactions.

**Comment by devongovett** [comment, marked as answer]
The main reason is because of the remove buttons within each tag. The listbox role does not allow interactive children within items, whereas grid does. See also: https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/layout-grids/

**Reply by JurajKruzliak** [reply]
Thanks for the reply. So that means that if I am using useTagGroup and never plan on using the remove tag, only the select functionality, it is a bit "overkill" to use grid, but it is not necessarily wrong, is that correct?
